src/lints/enum_variant_discriminant_changed.ron

```diff
@@ -0,0 +1,65 @@
+SemverQuery(
+    id: "enum_variant_discriminant_changed",
+    human_readable_name: "Public enum's variant had its discriminant changed from its previous value",
```

Comment:
This field is intended to be a name — it's printed in the headline of the CLI diagnostics we emit when breakage is found — so it should be quite short and it's okay if it doesn't have all the details.

Check out how these fields are used in the CLI ("what gets printed where") if you have a moment. I think that'll give you a better sense of what to put in the various fields going forward.
```suggestion
    human_readable_name: "enum variant had its discriminant change value",
```